.travis.yml

```diff
@@ -20,6 +22,13 @@ matrix:
       git:
         submodules: false
       env: JOB=SWIFTPM_DARWIN
+    - os: osx
+      osx_image: xcode9.4
+      before_script:
+        - gem install cocoapods
+      script:
+        - pod repo update
+        - pod spec lint --quick
```

Comment:
I think `pod lib lint` is more appropriate for library development which uses the files in the working directory

```sh
$ pod lib lint --help
Usage:

    $ pod lib lint

      Validates the Pod using the files in the working directory.
```

then https://github.com/Carthage/ReactiveTask/pull/111/files#r243798291 is not needed.

Reply:
@pepibumur It doesn't look like this has been addressed yet

Reply:
You uses `pod spec lint` not `pod lib lint` here.

Reply:
Done 👍